playground.py

```diff
@@ -0,0 +1,26 @@
+import graph.oval_graph
```

Comment:
Not every file has to be part of the repository. But if it is, it should have some description (e.g. as comments), and if it is a script, a proper command-line interface would be great too.